### PR TITLE
Only print ember version warning for apps

### PR DESCRIPTION
**Pull request by samselikoff**
Fixes https://github.com/rwjblue/ember-router-service-polyfill/issues/10

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -7,17 +7,18 @@ let hasBeenWarned = false;
...
 
-  included() {
+  included(includer) {
     this._super.included.apply(this, arguments);
 
     this._ensureThisImport();
 
     var checker = new VersionChecker(this);
     var emberVersion = checker.forEmber();
+    var includerIsApp = includer.name === this.project.name();
```
Comment (rwjblue) [inline index.js]: I wonder if we can use `project.isEmberCLIAddon()`?

https://github.com/ember-cli/ember-cli/blob/master/lib/models/project.js

**Comment by samselikoff** [comment]
Awesome API!! I tested it out locally with a new app and new addon and it works great 👍 

I'm getting Travis failures because of the `sandbox` issue, and forgetting what the fix was. Do you remember? https://travis-ci.org/rwjblue/ember-router-service-polyfill/jobs/389995110

**Comment by samselikoff** [comment]
@rwjblue should be all set

File: index.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ module.exports = {
 
     if (emberVersion.lt('2.16.0-alpha.1')) {
       this.import('vendor/ember-router-service-polyfill/index.js');
-    } else if (this.parent === this.project && !hasBeenWarned){
+    } else if (this.parent === this.project && !hasBeenWarned && this.project.isEmberCLIAddon()){
```
Comment (rwjblue) [inline index.js]: Hmm, wouldn't it be `!this.project.isEmberCLIAddon()`?  I think we want it to warn when this addon is directly included in apps but not addons...

**Review by rwjblue — CHANGES_REQUESTED** [review]

File: index.js (hunk 2 above)
Comment (samselikoff) [reply index.js]: Right..

**Review by rwjblue — APPROVED** [review]

**Comment by rwjblue** [comment]
Thank you!

 ember-router-service-polyfill 1.0.3 published 🎉

**Pull request merged**